plugins/backstage-plugin-env0/src/components/env0-variables-input/env0-variable-field.tsx

```diff
@@ -51,12 +51,30 @@ export const doesVariableValueMatchRegex = (variable: Variable) => {
   return regex.test(variable.value || '');
 };
 
+const getVariableHelperText = (
+  isRegexWrong: boolean,
+  isRequiredValueMissing: undefined | boolean,
+  variable: VariableFields & {
+    overwrites?: VariableFields;
```

Comment:
why do you need `overwrites`?

Reply:
the typing is redundant, dropped it
https://github.com/env0/env0-backstage-plugin/pull/75/commits/f8ce810f2e69f711022cdf6d88c4fdc32f7cc801